routes/api/team.js

```diff
@@ -127,6 +130,43 @@ module.exports = {
             Controllers.Team.showTeam
         );
 
+        /**
+         * @api {patch} /team/:hackerId Update a team's information. The team is specified by the hacker belonging to it.
+         * @apiName patchTeam
+         * @apiGroup Team
+         * @apiVersion 0.0.8
+         * 
+         * @apiParam (param) {ObjectId} hackerId a hacker's unique Id
+         * 
+         * @apiSuccess {String} message Success message
+         * @apiSuccess {Object} data Team object
+         * @apiSuccessExample {object} Success-Response: 
+         *      {
+                    "message": "Team update successful.", 
+                    "data": {...}
+                }
+
+         * @apiError {String} message Error message
+         * @apiError {Object} data Query input that caused the error.
+         * @apiErrorExample {object} Error-Response: 
+         *      {"message": "Team not found", "data": {teamId}}
+         */
+        teamRouter.route("/:hackerId").patch(
+            Middleware.Auth.ensureAuthenticated(),
+
+            Middleware.Auth.ensureAuthorized([Services.Hacker.findById]),
```

Comment:
Would be good to comment the design choice here for using `hackerId` as opposed to `teamId`.